src/ICeWall/doc/set_bump_options.md

```diff
@@ -0,0 +1,60 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump_options \
+        [-pitch pitch] \
+        [-spacing_to_edge spacing] \
```

Comment:
Changing the “spacing_to_edge” number does not make any change to the placement of the bumps

Reply:
Spacing to edge reflects a minimum value that is required, so number of bumps/placement may not change if the this minimum value is met.